api/sheets/get_property.md

```diff
@@ -6,41 +6,63 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the values of a given property for a node.
+# Retrieve property values for nodes
 
-## `=DCPROPERTY(dcids, property)`
+Given a list of nodes and a property label, returns values associated with the given property for each node.
 
-Given a property, together with a single DCID, a row of DCIDs, or a column of DCIDs, get the values of the given property for those DCIDs.
+## General information about this endpoint
 
-**Arguments**
-*    `dcids` - DCIDs to get the property for
-*    `property` - property to get
+**Formula**: `=DCPROPERTY(dcids, property)`
 
-**Returns**
+**Required arguments**:
 
-The properties of the DCIDs. For a single DCID, the result is a column of the given property's values for that DCID. For a row of DCIDs, the result is a matrix where each column contains the values of the given property for the DCID at the column's index. For a column of DCIDs, the result is a matrix where each row contains the values of the given property for the DCID at the row's index.
+*   `dcids`: A list of nodes to query, identified by their Data Commons identifiers.
+*   `property`: The property to query for.
 
-## Examples
+## Assembling the information you will need for a call to this method
 
-In this example, we find the `containedInPlace` of the Data Commons nodes for [California](https://datacommons.org/browser/geoId/06) and [Mountain View](https://datacommons.org/browser/geoId/0649670). The function call in the image uses a single call to get the names of both nodes, but it is equivalent to typing:
+Going into more detail on how to assemble the values for the required arguments:
 
-```
-=TRANSPOSE(DCPROPERTY("geoId/06", "containedInPlace"))
-```
+ - `dcids`: Data Commons uniquely identifies nodes by assigning them DCIDs, or Data Commons IDs. Your query will need to specify the DCIDs for the nodes of interest. More information about DCIDs is available in [the glossary](/glossary.html).
 
-and
+ - `property`: The property whose value you are interested in, such as "name" for the name of a node, or "typeOf" for the type of a node. If you aren't sure what properties are available for a particular DCID, you can use the [Data Commons graph browser](https://datacommons.org/browser/) to look up the DCID of interest and see what properties it is associated with.
 
-```
-=TRANSPOSE(DCPROPERTY("geoId/0649670", "containedInPlace"))
-```
+ **Returns**
```

Comment:
Does the returns section belong above the "assembling" section, along with the formula signature and arguments?

Reply:
We haven't usually done something like that (by analogy see https://docs.datacommons.org/api/rest/property_value.html), but if we want to move in that direction we can.

Reply:
could we start by moving this one up? this is more of a summary compared to the rest example.

Reply:
sure, no problem